src/gui/include/gui/gui.h

```diff
@@ -708,6 +708,8 @@ class Gui
   void showGui(const std::string& cmds = "",
                bool interactive = true,
                bool load_settings = true);
+  void iconify();
+  void deiconify();
```

Comment:
I don't find iconify an obvious term for minimum/maximize. If the help used the minimize/maximize I think that would be okay.

Reply:
I'm open to other terms.  Maximize suggests expanding to fill the whole screen to me.   Qt uses minimized (icon) / normal (deiconify) / maximized(expand window).   Gtk has iconify/deiconify.  We already have show/hide.  Minimize/unminimize?

Reply:
Minimized/unminimized works for me

Reply:
I prefer commands to be actions like show/hide so minimize/iconify sounds better.